pkg/acsengine/k8s_versions.go

```diff
@@ -143,12 +143,12 @@ var k8sComponentVersions = map[string]map[string]string{
 // KubeConfigs represents Docker images used for Kubernetes components based on Kubernetes versions (major.minor.patch)
 var KubeConfigs = map[string]map[string]string{
 	common.KubernetesVersion1Dot10Dot0: getK8sVersionComponents("1.10.0-beta.2", nil),
-	common.KubernetesVersion1Dot9Dot4:  getK8sVersionComponents("1.9.4", nil),
+	common.KubernetesVersion1Dot9Dot4:  getK8sVersionComponents("1.9.4", map[string]string{"windowszip": "v1.9.4-1int.zip"}),
```

Comment:
We actually don't need these overrides (though they don't hurt) because the default is to use the "-1int.zip" suffix

Reply:
Ah, didn't notice the override. Will revert.